posts/2022-04-15-libertydevops-gitjenkinshelmiks.adoc

```diff
@@ -56,14 +56,14 @@ image::/img/blog/pipeline-code-on-git.png[Pipeline code stored in a Jenkinsfile
 
 == Building the cloud-native Java application with Open Liberty on Jenkins
 
-Build your Dockerfile from the foundation code given below (gather the additional files mentioned in this code from Open Liberty support team).
+Build your Dockerfile from the foundation code given below. Additional files mentioned in the code can be downloaded from link:/extras/featureUtility.properties[featureUtility.properties] , link:/extras/server.xml[server.xml] 
 
 [source]
 ----
 FROM icr.io/appcafe/open-liberty:full-java11-openj9-ubi
 # Add Liberty server configuration including all necessary features
-COPY --chown=1001:0  server.xml /config/
-# Modify feature repository (optional)
+COPY --chown=1001:0 server.xml /config/
+# Modify feature repository (optional), this file contains the proxy settings to connect to private maven repository
 COPY --chown=1001:0 featureUtility.properties /opt/ol/wlp/etc/
```

Comment:
I don't think we need to include the proxy lines at all as it's optional and just complicates things.